QuizApp/Quiz/ViewModel/QuizViewModel.swift

```diff
@@ -2,6 +2,7 @@ import Combine
 
 class QuizViewModel {
 
+    @Published var errorOccurred: Bool = false
```

Comment:
Instead of `errorOccurred: Bool` You could have an enum: 

```
enum QuizError {

    case empty
    case serverError

}

 @Published var error: QuizError?

```
Try and change the logic using that enum. :)